pkg/sloop/server/internal/testFiles/testConfig.txt

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
unless there is a separate test that covers this - make sure that most/all of these are not the same as default values - want to ensure not accidentally getting the wrong config